.buildkite/commands/should-skip-job.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -eu
+
+# Usage: should-skip-job.sh --job-type [build|localization]
+# --job-type build: For jobs building an app binary, e.g. Build, Prototype Builds…
+#     Skip when changes are limited to documentation, tooling, and non-code files
+# --job-type localization: For jobs that only need to run when localization files have changed
+#     Skip when NO localization files have changed
+#
+# Return codes:
+# 0 - Job should be skipped (script also handles displaying the message and annotation)
+# 1 - Job should not be skipped
+# 15 - Error in script parameters
+
+COMMON_PATTERNS=(
+  "*.md"
+  "*.po"
+  "*.pot"
+  "*.txt"
+  ".gitignore"
+  "config/Version.xcconfig"
+  "fastlane/**"
+  "Gemfile"
+  "Gemfile.lock"
+)
+
+LOCALIZATION_PATTERNS=(
+  "**/*.strings"
+  "**/*.stringsdict"
+)
+
+# Check if arguments are valid
+if [ -z "${1:-}" ] || [ "$1" != "--job-type" ] || [ -z "${2:-}" ]; then
+  echo "Error: Must specify --job-type [build|localization]"
+  buildkite-agent step cancel
+  exit 15
+fi
+
+# Function to display skip message and create annotation
+show_skip_message() {
+  local job_type=$1
+  local message="Skipping ${BUILDKITE_LABEL:-Job} - no relevant files changed"
```

Comment:
After looking at the annotations in https://github.com/Automattic/pocket-casts-android/pull/4002 and here, I wonder if it might be best to use the past tense.

<img alt="image" src="https://github.com/user-attachments/assets/6ed4273e-314a-42e4-a861-d03164ca31d1" width=300/>

My reasoning is that "Skipping..." makes sense in the logs, but when one looks at the build annotation, the skipping has already happened.

```suggestion
  local message="Skipped ${BUILDKITE_LABEL:-Job} - no relevant files changed"
```

Reply:
Agreed, it makes sense to use "Skipped". Updated on 792a3555b62e47288c3908ec0aab956c2ca35700.